back-end/schemas/schemas.sql

```diff
@@ -129,29 +138,27 @@ CREATE TABLE IF NOT EXISTS Comment (
     CONSTRAINT fk_Comment_Post1 FOREIGN KEY (post_id) REFERENCES Post (post_id) ON DELETE NO ACTION ON UPDATE NO ACTION
 );
 -- -----------------------------------------------------
--- Table Follower
--- -----------------------------------------------------
-CREATE TABLE IF NOT EXISTS Follower (
-    follower_id INT NOT NULL AUTO_INCREMENT,
-    follower_counter INT NULL,
-    user_id INT UNIQUE NOT NULL,
-    PRIMARY KEY (follower_id),
-    UNIQUE INDEX follower_id_UNIQUE (follower_id ASC),
-    INDEX fk_Follower_User1_idx (user_id ASC),
-    CONSTRAINT fk_Follower_User1 FOREIGN KEY (user_id) REFERENCES User (user_id) ON DELETE NO ACTION ON UPDATE NO ACTION
+-- Table Follower_system
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS Follower_system (
+    follower_username VARCHAR(255) NOT NULL,
+    following_username VARCHAR(255) NOT NULL,
+    follows_id INT UNIQUE NOT NULL AUTO_INCREMENT,
+    create_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
+    updata_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
created_at would be a better name for the column, also since there is a default value it will never be null so you will have to remove the null

updated_at would be a better name for the column, and it is fine if it didn't have a default value.